.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           # doesn't declare support for Python 3
           shell: /bin/bash
           command: |
-            git clone -b master git://github.com/stackstorm-exchange/ci.git ~/ci
+            git clone -b master git@github.com:StackStorm-Exchange/ci.git ~/ci
```

Comment:
@cognifloyd Did we remove the CircleCI from all the Exchange? 

@bishopbm1 I guess you need to sync with the latest master as CircleCi was replaced with GH Actions.

Reply:
@bishopbm1 Would be also nice to replace the CircleCI status badge in this PR from the https://github.com/StackStorm-Exchange/stackstorm-bolt/blob/master/README.md to avoid confusion.

Reply:
Yea we will get them moved to GH actions. Everything started failing on our repos so we just updated to get the failures to stop while we transition

Reply:
@armab We should be good to go now!